instrumentation/opentelemetry-instrumentation-pymongo/src/opentelemetry/instrumentation/pymongo/__init__.py

```diff
@@ -126,6 +126,7 @@ def started(self, event: monitoring.CommandStartedEvent):
         statement = event.command_name
         if command:
             statement += " " + str(command)
+        collection = event.command.get(event.command_name)
```

Comment:
You can use command(line 123)

Reply:
According to the specs "The collection being accessed within the database stated in db.name."
so don't you need to use event.database_name instead of event.command_name?

Reply:
You are correct that this looks weird, but I have an explanation:

The `command` in line 123 is wrong, and i will change it as part of #1512 

About the collection name, that it just how it is stored inside the event, by the command name:
<img width="562" alt="image" src="https://user-images.githubusercontent.com/107620508/210521706-93b09aaa-1116-4ea0-a56d-1bd02b9b6989.png">